inginious/frontend/submission_manager.py

```diff
@@ -383,8 +386,13 @@ def get_feedback_from_submission(self, submission, only_feedback=False, show_eve
         """
         if only_feedback:
             submission = {"text": submission.get("text", None), "problems": dict(submission.get("problems", {}))}
+        
         if "text" in submission:
             submission["text"] = ParsableText(submission["text"], submission.get("response_type", "rst"), show_everything, translation).parse()
+
+        if "grader_results" in submission:
+            submission["text"] = ParsableText(submission, submission.get("response_type"), show_everything, translation).parse()
```

Comment:
add this only to ensure the logic flow, if grader_results in submmions, the only response type acceptable is dict, and if response_type is not found on the submission the default parse is rst, so better ensure this.
```suggestion
            submission["text"] = ParsableText(submission, submission.get("response_type","dict"), show_everything, translation).parse()
```